src/CMDParse.cpp

```diff
@@ -0,0 +1,57 @@
+#include "CMDParse.h"
+
+EERAModel::ArgumentParser::ArgumentParser(int argc, char** argv)
```

Comment:
For consistency with the other code, please declare the namespace in the implementation file and put the functions inside it:
```
namespace EERAModel {
...
```